app/views/invoices/show.html.erb

```diff
@@ -7,17 +7,24 @@
 <p>Customer Name: <%= @invoice.customer.first_name %> <%= @invoice.customer.last_name %></p>
 <table class="show-table">
   <tr>
+    <th>Item ID **FOR TESTING***</th>
```

Comment:
Just want to double check - did you intend to leave the "** FOR TESTING*** statement in here?

Reply:
Thanks for chatting with me about this @MWoodshop - I will leave it as is